src/index.ts

```diff
@@ -536,6 +538,14 @@ function getCacheName (sourceCode: string, fileName: string) {
 }
 
 /**
+ * Ensure the given cached content is valid by sniffing for a base64 encoded '}'
+ * at the end of the content, which should exist if there is a valid sourceMap present.
+ */
+function isValidCacheContent (content: string) {
+  return content.match(/(?:9|0=|Q==$)/)
```

Comment:
Please use `RegExp#test` instead of `String#match` (since we want `true` or `false`, not results). Also, you've got a small typo - need to put the `$` outside of the matching group - right now it's matching slightly incorrectly (any `9`, `0=` or `Q==` at the end). Since people can be iffy with regexps like this (we've already got one mistake just by a typo) we could change it to a string match like `content.slice(-1) === ...` and so on. Either way is good with me.